subscription-manager.spec

```diff
@@ -177,8 +177,10 @@ Requires: %{name} = %{version}-%{release}
 %if %use_gtk3
 Requires: pygobject3
 Requires: gtk3
+Requires: abattis-cantarell-fonts
 %else
 Requires: pygtk2 pygtk2-libglade
+Requires: dejavu-sans-fonts
```

Comment:
Sorry, I completely forgot about SuSE. I will use your first example. I consider it as not so ugly too :-)